test/unit/testUtil.js

```diff
@@ -21,33 +20,22 @@ define('testUtil', ['bluebird', 'narrativeConfig', 'json!/test/testConfig.json']
         return userId;
     }
 
-    function getCurrentNarrative() {
-        if (currentNarrative) {
-            return currentNarrative;
-        }
-        const token = getAuthToken();
-        // make a new workspace
-        // make a new narrative object and add it to the workspace
-        // add some data to the workspace
-    }
```

Comment:
got rid of this as `currentNarrative` was always null and the token was not used